msal/managed_identity.py

```diff
@@ -266,8 +280,7 @@ def acquire_token_for_client(
             and then a *claims challenge* will be returned by the target resource,
             as a `claims_challenge` directive in the `www-authenticate` header,
             even if the app developer did not opt in for the "CP1" client capability.
-            Upon receiving a `claims_challenge`, MSAL will skip a token cache read,
-            and will attempt to acquire a new token.
+            Upon receiving a `claims_challenge`, MSAL will attempt to acquire a new token.
```

Comment:
I think you should specifically say "skip the cache"

Reply:
Actually, I specifically changed that "skip the cache" into its current new form. Because "skip the cache" is an imprecise term (although we sometimes use it colloquially). Our implementation does need to look into the cache, in order to find a cached token (if any) and then calculate its hash.

Reply:
ok